docs/cFE Application Developers Guide.md

```diff
@@ -1080,21 +1080,17 @@ void SAMPLE_TaskMain(void)
         /*
```

Comment:
Can we rename this file so it does not use spaces?

Reply:
Please do as a separate issue